util/package.js

```diff
@@ -1,5 +1,20 @@
 'use strict';
 
+var allStars = require( 'all-stars' );
+var objectAssign = require( 'object-assign' );
+
+/**
+ * Check all-stars db for given person and assign to person if found.
+ *
+ * @param  {Object} person object representing author or maintainer
+ *
+ * @return {Object}        same object given, possibly modified or augmented
+ */
+function getAllStar( person ) {
+  // override properties from all-stars if available
+  return objectAssign( person, allStars( person ) );
+}
```

Comment:
Ok, did you want the "long" or "short" summary? Long looks like:

```
Rod Vagg <r@va.gg> (npm: rvagg, GitHub: rvagg, Twitter: rvagg)
```

While short looks like:

```
Rod Vagg <r@va.gg>
```

BTW, did you see [this comment about assigning](https://github.com/stefanjudis/credits/pull/8#discussion_r44888456)? Are you ok returning all that stuff?